app/src/components/course/index.jsx

```diff
@@ -1,20 +1,30 @@
 import { useUserContext } from "src/contexts/UserContext";
+import SearchBar from "src/components/SearchBar";
 import StaffCourse from "./staff/StaffCourse";
 import HRCourse from "./hr/HRCourse";
 
 function Course() {
   const { currentUserType } = useUserContext();
 
   switch (currentUserType) {
-    case "STAFF":
-      return <StaffCourse />;
     case "HR":
-      return <HRCourse />;
+      return (
+        <>
+          <SearchBar title='Search Courses' searchBarPlaceholder='Search by name, ID...' />
```

Comment:
Do we want the searchbar or can remove

Reply:
I think can keep just for standardisation sake